less/home-page.less

```diff
@@ -1 +1,94 @@
-// Home Content
\ No newline at end of file
+// Home Content
+
+.container {
+    width: 100%;
+    max-width: 880px;
+    font-size: 1.6rem;
+    margin: 0 auto;
+
+    .topImg {
+        width: 100%;
+        height: 240px;
+        margin-bottom: @marginBottom;
+    }
+    
+    h2 {
+        font-size: 2.4rem;
+        margin-bottom: @marginBottom;
+    }
+
+    p {
+        margin-bottom: @marginBottom;
+    }
+
+    .mainInfo {
+        display: flex;
+        flex-direction: column;
+        margin: 0 1%;
+        @media @phone {
+
+            margin: 0 2%;
+
+            .midImg2 {
+                display: none;
+            }
+
+            .midImg3 {
+                display: none;
+            }
+        }
+
+        .midImgAll {
+            margin-bottom: @marginBottom;
+            height: 180px;
+            .flex(space-between);
```

Comment:
I don't see a mixin called `.flex` 🔍 😕 Did this end up working the way you wanted?